data-explorer/kusto/management/mappings.md

```diff
@@ -65,7 +65,7 @@ Some of the data format mappings (Parquet, JSON and AVRO) support simple and use
 |-------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------|
 | PropertyBagArrayToDictionary  | Transforms JSON array of properties, such as `{events:[{"n1":"v1"},{"n2":"v2"}]}`, to dictionary and serializes it to valid JSON document, such as `{"n1":"v1","n2":"v2"}`. | Available for `JSON`, `Parquet`, `AVRO` and `ORC` mapping types. |
 | SourceLocation                | Name of the storage artifact that provided the data, type string (for example, the blob's "BaseUri" field).|Available for `CSV`, `JSON`, `Parquet`, `AVRO`, `ORC` and `W3CLOGFILE` mapping types. |
-| SourceLineNumber              | Offset relative to that storage artifact, type long (starting with '1' and incrementing per new record).   | Available in: `JSON`, `Parquet`, `AVRO`, `ORC` and `W3CLOGFILE` mapping types. |
+| SourceLineNumber              | Offset relative to that storage artifact, type long (starting with '1' and incrementing per new record).   | Available for `CSV`, `JSON`, `Parquet`, `AVRO`, `ORC` and `W3CLOGFILE` mapping types. |
```

Comment:
I have just verified that SourceLineNumber works for CSV mappings as well. Reading the docs I felt that I can't use it but was pleasantly surprised.

I also changed the format of "Available in:" to match other lines listing these values.